### DISCUSSION TITLE
how to remember the table width

### DISCUSSION THREAD

**Discussion by SunnyCheung-cp** [Milkdown/milkdown#1141] [General]
We can change the width of the table, but we can't remember and retrieve it
Can we get the width of a table?, or how can we disallow changing the width of a table?

https://github.com/Milkdown/milkdown/assets/105030733/715b9448-bae3-4752-9226-1bd953c2ec3c

**Comment by Saul-Mirone** [comment]
I think the only way to remember the table column is to save it in something like `localStorage`, otherwise you'll need to extend the markdown syntax to remember it. For example:

```markdown
| Header | Header |
|--------|--------|
| 100 | 200 |
| Cell | Cell |
| Cell | Cell | 
``` 

And when rendering, you'll hide the first line and take the value as width, which is possible but I personally won't recommend.

If you want to disable the resizer, you can filter [this plugin](https://github.com/Milkdown/milkdown/blob/main/packages/preset-gfm/src/plugin/column-resizing-plugin.ts).
